apm-sniffer/apm-sdk-plugin/kafka-plugin/src/main/java/org/apache/skywalking/apm/plugin/kafka/ConsumerConstructorInterceptor.java

```diff
@@ -22,18 +22,49 @@
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.EnhancedInstance;
 import org.apache.skywalking.apm.agent.core.plugin.interceptor.enhance.InstanceConstructorInterceptor;
 
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+
 /**
- *
+ * @author zhang xin, stalary
  **/
 public class ConsumerConstructorInterceptor implements InstanceConstructorInterceptor {
 
-    @Override
-    public void onConstruct(EnhancedInstance objInst, Object[] allArguments) {
-        ConsumerConfig config = (ConsumerConfig) allArguments[0];
-        // set the bootstrap server address
-        ConsumerEnhanceRequiredInfo requiredInfo = new ConsumerEnhanceRequiredInfo();
-        requiredInfo.setBrokerServers(config.getList("bootstrap.servers"));
-        requiredInfo.setGroupId(config.getString("group.id"));
+    @Override public void onConstruct(EnhancedInstance objInst, Object[] allArguments) {
+        ConsumerEnhanceRequiredInfo requiredInfo = resolveConsumerEnhanceRequiredInfo( allArguments[0] );
```

Comment:
Seems you're using a different code style, can you please import the [dedicated one](https://github.com/apache/skywalking/blob/master/codeStyle.xml) for SkyWalking to avoid unnecessary code changes?

Reply:
Working on setting up checkstyle in my IDE.

Reply:
checked all the modified with checkstyle rules